tasks/build-examples.js

```diff
@@ -10,7 +10,7 @@ var markupRegEx = /([^\/^\.]*)\.html$/;
 var cleanupJSRegEx = /.*(\/\/ NOCOMPILE|goog\.require\(.*\);)[\r\n]*/g;
 var requiresRegEx = /.*goog\.require\('(ol\.\S*)'\);/g;
 var isCssRegEx = /\.css$/;
-var isJsRegEx = /\.js$/;
+var isJsRegEx = /\.js(\?.*)?$/;
```

Comment:
See `examples/street-labels.html`. It requires a polyfill for `Set`, so it includes `https://cdn.polyfill.io/v2/polyfill.min.js?features=Set` as resource - that would not work with the previous regex.